src/app/components/tree-view/index.html

```diff
@@ -5,17 +5,21 @@
 
   <sky-demo-page-summary>
     <p>
-      The tree view module provides a hierarchical list view with multiple modes for selecting items.
+      The tree view module provides a hierarchical list view with multiple modes for selecting items in the list.
     </p>
-    <sky-alert alertType="warning">
-      This component is a release candidate. We do not plan to make any breaking changes the component before the official release, but such changes are possible.
-    </sky-alert>
+    <stache-include fileName="components/release-candidate-warning.html"></stache-include>
   </sky-demo-page-summary>
   
   <sky-demo-page-example>
     <sky-angular-tree-view-demo></sky-angular-tree-view-demo>
   </sky-demo-page-example>
 
+	<div class="sky-margin-stacked-content-section">
```

Comment:
What’s the deal with reordering items in the list? Messing around with the demo, I’ve been able to drag and drop list items, but it’s extremely temperamental and I can’t do it consistently. I don’t see a setting for this, so I don’t think it’s a property on our components. So does that mean it’s a feature of our components that is enabled by default? If so, should we have something about it in the design docs? Or is this inherited by the Angular components that we are building on? Also, if the feature is this fluky, should we try to disable it by default? Just clicking around, I was able to accidentally drag a cat to the dogs section in the demo, but I wasn’t able to drag it back to the correct section.

Reply:
I didn't think we were supporting drag & drop in this initial release, @Blackbaud-AlexKingman is this something built into the library that we need to disable?

Reply:
We haven't spent any time supporting it for this release. I could try and see if there's a way to prevent it, but I'd have to mess with the native 3rd-party library. The demo *shouldn't* be draggable, though. @blackbaud-johnly How did you get drag/drop enabled?

Reply:
I didn't do anything to enable drag/drop. I just noticed when I was clicking around selecting/deselecting items that I could click and drag. It's tempermental, but you can do it. I just tested it again, and was able to drag a couple cats to the dogs section. It's hit and miss on whether I can get items to be draggable though.
![image](https://user-images.githubusercontent.com/7850105/67317211-cca4a900-f4d7-11e9-83ce-4f8098075669.png)

Reply:
Whoa. You definitely should not be able to do that. I'll look into it.

What browser/OS are you using?

Reply:
Chrome on Windows 10.

Reply:
Reproduced. Filed a bug: https://github.com/blackbaud/skyux-angular-tree-component/issues/6

I'll get on it, thanks!

Reply:
Resolving since this is a bug with the component and not a docs issue.